test/extensions_test.dart

```diff
@@ -866,23 +866,6 @@ void main() {
         });
       });
     });
-    group('of nullable', () {
```

Comment:
We can leave the tests while we still ship it. Just ignore the lint!

Reply:
CI fails, not possible to ignore presumably

Reply:
You can ignore in the source code!

Reply:
bingo!